python/pyspark/ml/feature.py

```diff
@@ -443,6 +477,12 @@ class HashingTF(JavaTransformer, HasInputCol, HasOutputCol, HasNumFeatures):
     >>> params = {hashingTF.numFeatures: 5, hashingTF.outputCol: "vector"}
     >>> hashingTF.transform(df, params).head().vector
     SparseVector(5, {2: 1.0, 3: 1.0, 4: 1.0})
+    >>> hashingTFPath = temp_path + "/hashing-tf"
+    >>> hashingTF.save(hashingTFPath)
+    >>> loadedHashingTF = HashingTF.load(hashingTFPath)
+    >>> param = loadedHashingTF.getParam("numFeatures")
+    >>> loadedHashingTF.getOrDefault(param) == hashingTF.getOrDefault(param)
+    True
```

Comment:
Could you use ```getNumFeatures``` like other transformers in the doc test? It will make your test clean. ```HashingTF``` extends from ```HasNumFeatures```, so it has this method.